services/ark-api/ark-api/src/ark_api/api/v1/openai.py

```diff
@@ -165,7 +109,7 @@ async def chat_completions(request: ChatCompletionRequest) -> ChatCompletion:
     logger.info(f"Creating query for {target.type}/{target.name}")
 
     try:
-        async with with_ark_client("default", "v1alpha1") as ark_client:
+        async with with_ark_client(namespace, "v1alpha1") as ark_client:
```

Comment:
is v1alpha something we can extract into a constant i think it is used in many place